### DISCUSSION TITLE
Bug PTVO depuis dernière mise à jour Zigbee2Mqtt

### DISCUSSION THREAD

**Discussion by medline2c** [Koenkk/zigbee2mqtt#11248] [Devices]
Has anyone seen the same as me?

Yesterday, I updated and since then, all my PTVO modules are out of order... They no longer communicate with my Jeedom box and it is impossible to make any update in the module under penalty of error:

`**Error** Publish 'get' 'state' to 'zb_Inter CHB Parents' failed: 'Error: Read 0x00124b00089fd3f1/1 genOnOff(["onOff"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'UNSUP_CLUSTER_COMMAND')'`

`[MySQL] **Error code : 23000 (1062)**. Duplicate entry '236-Action' for key 'unique' : INSERT INTO cmd SETid = :id,logicalId = :logicalId,generic_type = :generic_type,eqType = :eqType,name = :name,order = :order,type = :type,subType = :subType,eqLogic_id = :eqLogic_id,isHistorized = :isHistorized,unite = :unite,configuration = :configuration,template = :template,display = :display,value = :value,isVisible = :isVisible,alert = :alert`

Any ideas?...how to revert to the previous version which was working fine?...

**Comment by Hedda** [comment]


**Reply by medline2c** [reply]
Sorry, with the Google Translate Chrome extensions, I didn't realize I was in machine translation... that explains it.

I'm going to edit my post to put it in English now...

Thks
